xonsh/shell.py

```diff
@@ -60,6 +61,10 @@ def __init__(self, ctx=None, shell_type=None, config=None, rc=None,
                 warn('prompt-toolkit version < v1.0.0 is not supported. '
                      'Please update prompt-toolkit. Using readline instead.')
                 shell_type = 'readline'
+            elif not HAS_PYGMENTS:
+                warn('prompt_toolkit is available but requires pygments.  '
```

Comment:
Yeah, that's what got me started on this.  Should it instead warn that "xonsh requires pygments alongside prompt_toolkit"?